src/OpenLoco/Windows/PromptOkCancelWindow.cpp

```diff
@@ -97,6 +98,17 @@ namespace OpenLoco::Ui::Windows
         return _result != 0;
     }
 
+    // 0x00447125
+    void promptOkCancelInput(uint32_t charCode, uint32_t keyCode)
+    {
+        auto window = WindowManager::find(WindowType::confirmationPrompt);
+        if (window == nullptr)
+            return;
+
+        if (charCode == VK_ESCAPE)
+            WindowManager::close(window->type);
```

Comment:
Perhaps better stick closer to the original:
```suggestion
            window->callOnMouseUp(widx::close);
```